pylot/perception/detection/detection_operator.py

```diff
@@ -171,3 +124,10 @@ def __run_model(self, image_np):
         res_boxes = boxes[0][:num_detections]
         res_scores = scores[0][:num_detections]
         return num_detections, res_boxes, res_scores, res_classes
+
+    def destroy(self):
+        self._logger.warn('destroying {}'.format(self.config.name))
+        # Sending top watermark because the operator is not flowing
+        # watermarks.
+        # self._obstacles_stream.send(
+        #     erdos.WatermarkMessage(erdos.Timestamp(is_top=True)))
```

Comment:
Could you delete this? The new version of ERDOS should automatically send top watermarks after `destroy()` finishes running.